jwst/datamodels/tests/test_models.py

```diff
@@ -615,12 +615,17 @@ def test_info():
 
 
 def test_validate_on_read():
+    """ Test for proper validation error
+
+    Note: The FITS file is opened separately in order to properly close
+    the file.
+    """
     schema = ImageModel((10, 10))._schema.copy()
     schema['properties']['meta']['properties']['calibration_software_version']['fits_required'] = True
 
-    with pytest.raises(jsonschema.ValidationError):
-        with ImageModel(FITS_FILE, schema=schema, strict_validation=True):
-            pass
+    with fits.open(FITS_FILE) as hduls:
+        with pytest.raises(jsonschema.ValidationError):
+            ImageModel(hduls, schema=schema, strict_validation=True)
```

Comment:
Why doesn't `with ImageModel():` context manager close it?  It should.

Reply:
That behavoir would be bad form. Nothing, if passed on open resource, should inherently close that resource. `DataModel` should not presume intent. The application may be doing something else with that open resource.